Can't resolve babel-runtime - doesn't work with Babel v7

Building a project that includes `emoji-mart` but not `babel-runtime` fails:

```
ERROR in ./node_modules/emoji-mart/dist-es/utils/index.js
Module not found: Error: Can't resolve 'babel-runtime/core-js/object/keys' in '[...]/node_modules/emoji-mart/dist-es/utils'

ERROR in ./node_modules/emoji-mart/dist-es/components/skins.js
Module not found: Error: Can't resolve 'babel-runtime/helpers/classCallCheck' in '[...]/node_modules/emoji-mart/dist-es/components'

...etc.
```

I see in `.babelrc` that some Babel imports are aliased to polyfills by the `module-resolver` plugin, but not all of them, so the dist files depend on packages that aren't necessarily installed.

For example, this can happen if a project is using Babel v7, which has different package names.

Perhaps the project should explicitly include Babel in `dependencies` instead of `devDependencies`, or else make sure all the imports are covered by polyfills?

--- a/.babelrc
+++ b/.babelrc
@@ -34,7 +34,9 @@
           "babel-runtime/helpers/extends": "./src/polyfills/extends",
           "babel-runtime/helpers/inherits": "./src/polyfills/inherits",
           "babel-runtime/helpers/createClass": "./src/polyfills/createClass",
-          "babel-runtime/helpers/possibleConstructorReturn": "./src/polyfills/possibleConstructorReturn"
+          "babel-runtime/helpers/possibleConstructorReturn": "./src/polyfills/possibleConstructorReturn",
+          "babel-runtime/helpers/classCallCheck": "./src/polyfills/classCallCheck",
+          "babel-runtime/core-js/object/keys": "./src/polyfills/keys"
         }
       }
     ]

--- a/src/polyfills/classCallCheck.js
+++ b/src/polyfills/classCallCheck.js
@@ -1,0 +1,5 @@
+export default function(instance, Constructor) {
+  if (!(instance instanceof Constructor)) {
+    throw new TypeError('Cannot call a class as a function')
+  }
+}

--- a/src/polyfills/keys.js
+++ b/src/polyfills/keys.js
@@ -1,0 +1,38 @@
+// From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
+var hasOwnProperty = Object.prototype.hasOwnProperty,
+  hasDontEnumBug = !{ toString: null }.propertyIsEnumerable('toString'),
+  dontEnums = [
+    'toString',
+    'toLocaleString',
+    'valueOf',
+    'hasOwnProperty',
+    'isPrototypeOf',
+    'propertyIsEnumerable',
+    'constructor',
+  ],
+  dontEnumsLength = dontEnums.length
+
+export default function(obj) {
+  if (typeof obj !== 'function' && (typeof obj !== 'object' || obj === null)) {
+    throw new TypeError('Object.keys called on non-object')
+  }
+
+  var result = [],
+    prop,
+    i
+
+  for (prop in obj) {
+    if (hasOwnProperty.call(obj, prop)) {
+      result.push(prop)
+    }
+  }
+
+  if (hasDontEnumBug) {
+    for (i = 0; i < dontEnumsLength; i++) {
+      if (hasOwnProperty.call(obj, dontEnums[i])) {
+        result.push(dontEnums[i])
+      }
+    }
+  }
+  return result
+}

